main.go

```diff
@@ -25,15 +29,48 @@ import (
 )
 
 func main() {
-	err := builder.APIServer.
-		// +kubebuilder:scaffold:resource-register
-		WithResourceAndHandler(&appcatv1.AppCat{}, appcat.New()).
-		WithResourceAndHandler(&appcatv1.VSHNPostgresBackup{}, postgres.New()).
-		WithoutEtcd().
+
+	var appcatEnabled bool = true
+	var vshnEnabled bool = false
+	var err error
+
+	if os.Getenv("APPCAT_HANDLER_ENABLED") != "" {
```

Comment:
You don't need this check anymore. If it's `""` `PraseBool` will just return an error.